models/app.model.js

```diff
@@ -7,15 +7,18 @@ exports.readTopics = () => {
   }
 
   exports.fetchArticleById = (article_id) =>{
-    if(isNaN(parseInt(article_id))){
-      return Promise.reject({ status: 400, msg: "Bad Request" });
-    }
     return db
     .query(`SELECT * FROM articles WHERE article_id=$1`, [article_id])
     .then(({ rows }) => {
     if(rows.length === 0){
-      return Promise.reject({code: "INVALID_ID", msg: "Article Doesn't Exist"})
+      return Promise.reject({status: 404, msg: "Article Doesn't Exist"})
     }
     return rows[0];
   })
+}
+
+exports.selectArticles = () => {
+  return db.query(`SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id)::int AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC`).then(({rows}) => {
```

Comment:
`COUNT(comments.article_id)::int` <- Really nice use of `::` here, do you know the technical term of what this is doing?